tardis/simulation/base.py

```diff
@@ -4,10 +4,11 @@
 import pandas as pd
 from astropy import units as u
 from collections import OrderedDict
-
+import h5py
```

Comment:
As of now, `pandas.HDFStore()` does not support iterating over hierarchically , while `h5py` does.
The issue is still opened there . [Link](https://github.com/pandas-dev/pandas/issues/10143).

So, I am using `h5py` only for transversing HDF file , and for reading particular attribute (like `time_explosion`) , I am using `pd.HDFStore()`